Extension/src/LanguageServer/copilotCompletionContextProvider.ts

```diff
@@ -247,7 +281,12 @@ export class CopilotCompletionContextProvider implements ContextResolver<CodeSni
             const featureFlag: CopilotCompletionContextFeatures | undefined = await this.getEnabledFeatureFlag(context);
             telemetry.addRequestMetadata(context.documentContext.uri, context.documentContext.offset,
                 context.completionId, context.documentContext.languageId, { featureFlag, timeBudgetFactor, maxCaretDistance });
-            if (featureFlag === undefined) { return []; }
+            if (featureFlag === undefined) { throw new Error("No feature flag set!"); }
+            // eslint-disable-next-line no-bitwise
+            if (featureFlag & CopilotCompletionContextFeatures.AlwaysFallbackToSimilarFiles) {
+                telemetry.addCompletionContextKind(CopilotCompletionKind.AlwaysFallbackToSimilarFiles);
+                throw new Error("Enable SimilarFiles");

```

Comment:
Same as the 0 flags case -- are you sure you want to log this case as an error in telemetry?

Reply:
Users who view the Toggle Developer Tool may think it's an actual error:

![image](https://github.com/user-attachments/assets/c3751b71-7bcb-476c-84f6-e231f2c54f4c)

Reply:
yes, it should be logged in telemetry, but not necessarily in log. The case should not be possible, i.e. flags must always be set.
Basically the `resolve()` function is NOT called if the VSCode instance (or user) is not part of the experiment. In your case it is being called because you have it enabled in your settings.